db/migrate/20200610181353_add_opt_out_of_cookies_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddOptOutOfCookiesToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :opt_out_of_cookies, :boolean, default: false
```

Comment:
I have come to learn that it can — sometimes — be useful to _not_ add a default value to boolean flags unless the default is `true`. That's because, for example, **possibly** with this flag, we may want to query for users whose flag has not been updated and whose flag has not been explicitly been set to to `false`. 

`nil` is falsey anyway, so that's why not setting a default to `false` would work :).

Just something to consider.

Reply:
We want this to explicitly be false unless someone sets it to true in the admin which is why I added the default value.